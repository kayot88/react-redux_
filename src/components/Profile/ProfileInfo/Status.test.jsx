

describe('Status compoent testing', () => {
  test('status should be rendered', () => {
    
  });
});