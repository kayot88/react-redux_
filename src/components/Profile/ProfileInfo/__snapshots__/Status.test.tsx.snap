// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status component should snapshot component 1`] = `
<div>
  <div>
    <span
      data-testid="span"
      onClick={[Function]}
    >
      Enter status here
      ----
    </span>
  </div>
</div>
`;
